src/finch/codegen/numpy_buffer.py

```diff
@@ -70,8 +81,17 @@ def deserialize_from_c(self, c_buffer):
         Update this buffer based on how the C call modified the NumpyCBuffer structure.
         """
 
+    def serialize_to_numba(self):
+        return self.arr
+
+
+class NumbaBufferFormat:
+    @staticmethod
+    def full_name():
+        return "numpy.ndarray"
```

Comment:
is this still needed? If it's needed, does it make sense to give it a more numba-related name? I'm guessing the purpose is to emit the name of the class returned by `serialize_to_numba`?

Reply:
Correct, it's needed for code generation to have `a: numpy.ndarray`. I renamed it to `def numba_name()` and moved the definition of the class to `numba_backend.py`.